specification/appendix-c/index.md

```diff
@@ -17,3 +17,7 @@ There is a dedicated [OpenFeature working group](https://github.com/open-feature
 - Reference implementations of the protocol
 
 If you are interested in this initiative, you can join the [`#openfeature-remote-evaluation-protocol`](https://cloud-native.slack.com/archives/C066A48LK35) Slack channel on the [CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) workspace.
+
+## API Specification
+
+View the full [OFREP API Specification](/docs/reference/other-technologies/ofrep/openapi) with interactive documentation.
```

Comment:
This is causing a lint error because the link breaks locally.

You could somehow exclude it from the markdown check, OR you could change it to https://openfeature.dev/docs/reference/other-technologies/ofrep/openapi, which currently returns a 204 (lint should pass) and will even work if people are browsing the spec in the repo.